lib/util/parse.js

```diff
@@ -137,9 +137,7 @@ async function parseGethParams (json) {
   }
   params.hardforks = hardforks.map(name => ({
     name: name,
-    block: name === 'chainstart' ? 0 : json.config[forkMap[name]] || null,
-    consensus: json.config.clique ? 'poa' : 'pow',
```

Comment:
Was a bit unsure about the `consensus` part, since the PoA information is directly extracted from a geth configuration file, but the information is nowhere used within the client so I think it should be safe to remove. Can also be re-added later if necessary, eventually in another context then.